src/test/ui/consts/const_let_eq.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(const_let, const_fn)]
+#![feature(const_fn)]
```

Comment:
I wrote everything in this file specifically to be good without `const_fn` once `const_let` stabilizes; what requires `const_fn` here?